protected/views/site/help.php

```diff
@@ -1,362 +1,929 @@
+                        <p>Available <strong>attribute_name</strong> to search include:</p>
+                        <dl class="help-description-list">
+                            <div class="help-definition-container">
+                                <dt><strong>taxno</strong></dt>
+                                <dd>Taxonomic ID (NCBI)</dd>
+                            </div>
+
+                            <div class="help-definition-container">
+                                <dt><strong>taxname</strong></dt>
+                                <dd>species name (nb must exact spelling, no synonyms searched)</dd>
+                            </div>
+
+                            <div class="help-definition-container">
+                                <dt><strong>author</strong></dt>
+                                <dd>restricts search to the author table</dd>
+                            </div>
+
+                            <div class="help-definition-container">
+                                <dt><strong>datasettype</strong></dt>
+                                <dd>restricts search to the types of datasets, e.g. metagenomic, genomic, transcriptomic etc.</dd>
+                            </div>
+
+                            <div class="help-definition-container">
+                                <dt><strong>manuscript</strong></dt>
+                                <dd>restricts search to the manuscript ID associated with <span class="text-italic">GigaDB</span> dataset(s) e.g. <span>search?manuscript=10.1186/2047-217X-3-21</span></dd>
+                            </div>
+
+                            <div class="help-definition-container">
+                                <dt><strong>project</strong></dt>
+                                <dd>restricts search to the project name, e.g. Genome 10K</dd>
+                            </div>
+                        </dl>
+                        <p>eg. <span>..../search?taxno=9606</span></p>
+                        <p>To specify results to be returned are ONLY a particular level of data, add the phrase <strong><span>&amp;results=dataset</span></strong>, or file or sample:<br />e.g. <a href="http://gigadb.org/api/search?project=Genome%2010K&amp;result=sample" target="_blank">http://gigadb.org/api/search?project=Genome%2010K&amp;result=sample</a></p>
+                        <p>NB - the search still looks everywhere, but the results returned are only those samples that are in datasets that are found by the search.</p>
+                        <p><strong>Default results are "dataset" only.</strong></p>
+                    </section>
+
+                    <hr class="dashed">
+
+                    <section aria-labelledby="examplesTitle" class="m-0">
+                        <h2 class="page-subtitle" id="examplesTitle">Examples</h2>
+
+                        <ol class="help-examples-list">
```

Comment:
this used to be a sequence of `<p>` numbered by hand, replaced it by a more semantic `<ol>`